packages/web-components/src/components/image-with-caption/image-with-caption.ts

```diff
@@ -12,10 +12,11 @@ import settings from 'carbon-components/es/globals/js/settings';
 import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
 import on from 'carbon-components/es/globals/js/misc/on';
 import '../image/image';
+import '../modal/modal';
+import '../modal/modal-close-button';
```

Comment:
@asudoh should the expressive modal changes go in first?

Reply:
@jeffchew We can get one that's merged later in sync with another.